src/Symfony/Component/Security/Core/Resources/translations/security.hr.xlf

```diff
@@ -70,6 +70,14 @@
                 <source>Invalid or expired login link.</source>
                 <target>Link za prijavu je isteako ili je neispravan.</target>
             </trans-unit>
+            <trans-unit id="19">
+                <source>Too many failed login attempts, please try again in %minutes% minute.</source>
+                <target>Previše neuspjelih pokušaja prijave, molim pokušajte ponovo za %minutes% minutu.</target>
+            </trans-unit>
+            <trans-unit id="20">
+                <source>Too many failed login attempts, please try again in %minutes% minutes.</source>
+                <target>Previše neuspjelih pokušaja prijave, molim pokušajte ponovo za %minutes% minuta.</target>
```

Comment:
Note: this is actually wrong as it should be 2..4 minut**e**, 5..20 minut**a**, 21 minut**u**, 22..24 minut**e**, etc., but I don't think that's currently possible.

Reply:
IIRC we had a PR with this "problem" one or two months ago. Maybe you can double check the recent closed/open PRs.

@Nyholm do you remember?

Reply:
I think you are referring to this PR: https://github.com/symfony/symfony/pull/41097

@HypeMC, do you see how they achieved that for Lithuanian?

Reply:
Exactly, many thanks Tobias 🌴

Reply:
@OskarStark @Nyholm Works like a charm, thanks for the help.

Reply:
Very welcome, thanks for another great contribution 🥰